src/main/resources/application.properties

```diff
@@ -182,5 +182,6 @@ portal.config.ssoLoginAllowed=false
 portal.config.defaultLanguage=en
 portal.config.testInstance=false
 portal.config.sendAppInstanceFailureEmails=false
+portal.config.showDomainRegistrationSelector=true
```

Comment:
Also add this property in application.properties template file in docker directory